src/utils/sortOrders.test.ts

```diff
@@ -1,4 +1,43 @@
-import {sortByItemCount} from './sortOrders';
+import {getSortFunction, sortByDate, sortByItemCount, sortOrders, sortTypes} from './sortOrders';
+
+describe('sortOrders function', () => {
+	it('count sort type', () => {
+		const orders = [
+			{ items: ['item1', 'item2', 'item3'] },
+			{ items: ['1', '2'] },
+		];
+		const ordersCopy = [...orders];
+		sortOrders(orders, sortByItemCount);
+		expect(orders).not.toEqual(ordersCopy);
```

Comment:
Мне кажется, можно
```suggestion
		expect(orders).not.toStrictEqual(ordersCopy);
```

([документация](https://jestjs.io/ru/docs/expect#tostrictequalvalue))

Но могу ошибаться!